docs/docs/manual/installing.md

```diff
@@ -93,16 +93,16 @@ import TabItem from '@theme/TabItem';
 
 <TabItem value="win">
 
-1. On Windows 10, click the Windows start menu button, type "env," and look at the search results. Click “Edit the system environment variables.” (If you are using an earlier version of Windows, use the “Search” or “Search programs and files” box in the start menu.)
+1. On Windows 10, click the Windows start menu button, type `env`, and look at the search results. Click `Edit the system environment variables`. (If you are using an earlier version of Windows, use the `Search` or `Search programs and files` box in the start menu.)
```

Comment:
**Edit the system environment variables** is a button on Windows OS, not a button or value in OpenRefine.

@allanaaa what do we do for buttons and elements on the OS's ?  Perhaps we  can borrow conventions that others do, like what MS does? https://code.visualstudio.com/docs/editor/versioncontrol

Convention Suggestions:
1. Code and Values can use 1 (inline code) or 3 backticks (codeblock)
2. Use a nice style for keyboard commands (or just **Bold** for now since we don't have many)
3. For OpenRefine buttons we can use the menuItem class
4. For OS buttons we can use a simple **Bold**

Reply:
I would suggest the following:
1. For menuItem class, taking inspiration from MS docs,
![Screenshot 2020-09-11 at 15 10 45](https://user-images.githubusercontent.com/29340368/92929533-ff81c380-f440-11ea-8297-7298c3a210ed.png)

2. I also feel that the convention for the OS elements and as well as the commands can be Bold. 

3. I suggest code, that is the inline backticks to have the following style 
![Screenshot 2020-09-11 at 15 21 59](https://user-images.githubusercontent.com/29340368/92931232-6d2eef00-f443-11ea-8b60-dc2f4a02a70b.png)

thoughts @allanaaa , @thadguidry , @wetneb  ??

Reply:
I like it, but can we make the menuItem class background a bit lighter toned blue color?

#bed4f7

Reply:
using #bed4f7
![Screenshot 2020-09-11 at 17 16 36](https://user-images.githubusercontent.com/29340368/92943051-960ab080-f452-11ea-9955-478ac11c0536.png)

Reply:
@sjathin Hmm, lighten it 2 shades perhaps (slightly lighter blue from that then should be fine)